packages/core/src/components/ui/Logo/Logo.tsx

```diff
@@ -1,5 +1,26 @@
-function Logo() {
-  return <span data-fs-logo role="img" aria-label="BaseStore logo" />
+import { Image } from 'src/components/ui/Image'
+import type { ImageProps } from 'src/components/ui/Image'
+
+interface LogoProps {
+  alt: string
+  src: string
+  loading?: ImageProps['loading']
+}
+
+function Logo({ alt, src, loading = 'lazy' }: LogoProps) {
+  return (
+    <div data-fs-logo>
```

Comment:
I think now we need to add a new doc to explain that the logo should be added on the CMS, cuz it's an element that it's part of the branding, but it's not a configuration of the theme anymore.